widget_upload.py

```diff
@@ -11,30 +11,40 @@
 import argparse
 import demisto
 
+
 def p(what):
     if verbose:
         print(what)
 
+
 def options_handler():
-    parser = argparse.ArgumentParser(description='Upload a new widget to Demisto')
-    parser.add_argument('-k', '--key', help='The API key to access the server', required=True)
-    parser.add_argument('-s', '--server', help='The server URL to connect to', required=True)
-    parser.add_argument('-q', '--quiet', action='store_false', dest='verbose', default=True, help="no extra prints")
-    parser.add_argument('-w', '--widget', help='The new widget JSON file', required=True)
+    parser = argparse.ArgumentParser(
+        description='Upload a new widget to Demisto')
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-s', '--server', help='The server URL to connect to', required=True)
+    parser.add_argument('-q', '--quiet', action='store_false',
+                        dest='verbose', default=True, help="no extra prints")
+    parser.add_argument(
+        '-w', '--widget', help='The new widget JSON file', required=True)
```

Comment:
Positional argument instead?

Reply:
use positional arguments instead of required=True.(e2beb08)